src/js/pages/jobs/JobDetailPage.js

```diff
@@ -329,10 +331,7 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
       onItemSelect: this.handleEditButtonClick
     });
 
-    actions.push({
```

Comment:
because the store method isn't used anymore, you should be able to remove `handleRunNowButtonClick` from `JobDetailPage`